go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/launchdarkly/ldcli
 
-go 1.20
+go 1.21
```

Comment:
Although it's not strictly true that we require 1.21, since we're building with 1.21 and 1.22 we can't claim this. See my other comment for an alternative.